index.html

```diff
@@ -19,9 +19,9 @@
     <header>
       <div class="container header">
         <div class="row">
-          <div class="column-full">
-            <h1>Code Journal</h1>
-          </div>
+          <div class="column-full"></div>
+          <h1 class="heading">Code Journal</h1>
+          <a class="show-entries">Entries</a>
```

Comment:
Needs an `href`. You can do `href="#". This will force the screen to scroll to top.